client/snippets/snippets.json

```diff
@@ -39,5 +52,267 @@
+        "description": "Verb"
+    },
+    "For Each with Assembly ID": {
+        "prefix": "\\foreach",
+        "body": [
+            "\\foreach(${1:var}, ${2:in}, ${3:id})",
+            "\t$0",
+            "\\endForeach"
+        ],
+        "description": "“For each” loop with assembly ID"
+    },
+    "For Each": {
+        "prefix": "\\foreach",
+        "body": [
+            "\\foreach(${1:var}, ${2:in})",
+            "\t$0",
+            "\\endForeach"
+        ],
+        "description": "“For each” loop"
+    },
+    "Static List": {
+        "prefix": "\\beginList",
+        "body": [
+            "\\beginList",
+            "\t\\nextItem",
+            "\t\t${1:xxx}",
+            "\t",
+            "\t\\nextItem",
+            "\t\t${2:yyy}",
+            "\t$0",
+            "\\endList"
+        ],
+        "description": "Static List"
+    },
+    "Static List Item": {
+        "prefix": "\\nextItem",
+        "body": [
+            "\\nextItem",
+            "\t$0"
+        ],
+        "description": "An item in a static list"
+    },
+    "Synonyms": {
+        "prefix": "\\beginSynonym",
+        "body": [
+            "\\beginSynonym",
+            "\t\\choice",
+            "\t\t${1:xxx}",
+            "\t",
+            "\t\\choice",
+            "\t\t${2:yyy}",
+            "\t$0",
+            "\\endSynonym"
+        ],
+        "description": "List of alternatives"
+    },
+    "Synonyms (shorthand)": {
+        "prefix": "synonyms",
+        "body": [
+            "\\beginSynonym",
+            "\t\\choice",
+            "\t\t${1:xxx}",
+            "\t",
+            "\t\\choice",
+            "\t\t${2:yyy}",
+            "\t$0",
+            "\\endSynonym"
+        ],
+        "description": "List of alternatives"
+    },
+    "Synonym Choice": {
+        "prefix": "\\choice",
+        "body": [
+            "\\choice",
+            "\t$0"
+        ],
+        "description": "An alternative in a list of synonyms"
+    },
+    "Switch statement": {
+        "prefix": "\\switch",
+        "body": [
+            "\\switch(${1:expr})",
+            "\t\\case(${2:val_1})",
+            "\t\t${3:code_1}",
+            "\t",
+            "\t\\case(${4:val_2})",
+            "\t\t${5:code_2}",
+            "\t",
+            "\t\\case(${6:val_3})",
+            "\t\t${7:code_3}",
+            "\t$0",
+            "\\endSwitch"
+        ],
+        "description": "Handle values differently"
+    },
+    "Switch case item": {
```

Comment:
I would just copy-paste my code. Not needed, I think.

Reply:
Same arguments as for synonym choices. Plus, all these small things are very unobtrusive and should do no harm whatsoever.